domain/modeldefaults/bootstrap/bootstrap.go

```diff
@@ -8,21 +8,21 @@ import (
 
 	"github.com/juju/juju/domain/modeldefaults"
 	"github.com/juju/juju/domain/modeldefaults/service"
+	"github.com/juju/juju/domain/modeldefaults/state"
 	"github.com/juju/juju/environs/config"
 )
 
 // ModelDefaultsProvider is a bootstrap helper that wraps the raw config values
 // passed in during bootstrap into a model default provider interface to be used
 // when persisting initial model config. Config passed to this func can be nil.
 func ModelDefaultsProvider(
-	configDefaults map[string]any,
```

Comment:
Why did we originally have this arg? And why remove the arg now instead of passing in the defaults at the call site?

Reply:
The design has changed a little since the first iteration of this signature. It was just left over and not updated hence the problem you ran into.